ansible/roles/swap/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
 - name: Set swappiness
   sysctl:
     name: vm.swappiness
-    value: 0
+    value: 1
```

Comment:
@millerdev should this change get pulled out to master and deployed to all environments?